specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/managedClusters.json

```diff
@@ -2431,76 +2160,81 @@
         },
         "tags": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
-          "description": "Agent pool tags to be persisted on the agent pool virtual machine scale set."
+          "description": "The tags to be persisted on the agent pool virtual machine scale set."
         },
         "nodeLabels": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
-          "description": "Agent pool node labels to be persisted across all nodes in agent pool."
+          "description": "The node labels to be persisted across all nodes in agent pool."
         },
         "nodeTaints": {
           "type": "array",
           "items": {
             "type": "string"
           },
-          "description": "Taints added to new nodes during node pool create and scale. For example, key=value:NoSchedule."
+          "description": "The taints added to new nodes during node pool create and scale. For example, key=value:NoSchedule."
         },
         "proximityPlacementGroupID": {
-          "$ref": "#/definitions/ProximityPlacementGroupID",
+          "type": "string",
           "description": "The ID for Proximity Placement Group."
         },
         "kubeletConfig": {
           "$ref": "#/definitions/KubeletConfig",
-          "description": "KubeletConfig specifies the configuration of kubelet on agent nodes."
+          "description": "The Kubelet configuration on the agent pool nodes."
         },
         "linuxOSConfig": {
           "$ref": "#/definitions/LinuxOSConfig",
-          "description": "LinuxOSConfig specifies the OS configuration of linux agent nodes."
+          "description": "The OS configuration of Linux agent nodes."
         },
         "enableEncryptionAtHost": {
           "type": "boolean",
-          "description": "Whether to enable EncryptionAtHost"
+          "title": "Whether to enable host based OS and data drive encryption.",
+          "description": "This is only supported on certain VM sizes and in certain Azure regions. For more information, see: https://docs.microsoft.com/azure/aks/enable-host-encryption"
         },
         "enableUltraSSD": {
           "type": "boolean",
           "description": "Whether to enable UltraSSD"
         },
         "enableFIPS": {
           "type": "boolean",
-          "description": "Whether to use FIPS enabled OS"
+          "description": "Whether to use a FIPS enabled OS."
         },
         "gpuInstanceProfile": {
           "$ref": "#/definitions/GPUInstanceProfile",
-          "description": "GPUInstanceProfile to be used to specify GPU MIG instance profile for supported GPU VM SKU. Supported values are MIG1g, MIG2g, MIG3g, MIG4g and MIG7g."
+          "description": "GPUInstanceProfile to be used to specify GPU MIG instance profile for supported GPU VM SKU."
```

Comment:
This is a lot of abbreviations. When can this be set? What is it for ? What does it enable? Needs significant clarification.

Reply:
Usually API argument doc is a brief introduction. It doesn't cover full details of the usage, which will be covered by feature doc.  But the supported values should be kept to make it easier to use.

@qinchen352 FYI.

Reply:
@xuto2 - The type being `ref`'ed is an enum with these values. As you can see in the [REST API Docs](https://docs.microsoft.com/en-us/rest/api/aks/managed-clusters/create-or-update#gpuinstanceprofile-) and in the [Python SDK](https://github.com/Azure/azure-sdk-for-python/blob/f1a6b32afa460ae15f3b36b85ba240baafd4a1ad/sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2021_03_01/models/_models_py3.py#L165) when something is defined as an enum it already gets a list of supported values following it.

In the case of the most SDKs (including Python) this results in the awkward:
> GPUInstanceProfile to be used to specify GPU MIG instance profile
     for supported GPU VM SKU. Supported values are MIG1g, MIG2g, MIG3g, MIG4g and MIG7g. Possible
     values include: "MIG1g", "MIG2g", "MIG3g", "MIG4g", "MIG7g".
     
 Where we say the same thing twice in a row.